### ISSUE TITLE
HFC doesn't compile/link anymore with stable Mono 4.6.1 - cannot find -lmono-2.0

### ISSUE THREAD

**Issue by derFunk** [xplicit/HyperFastCgi#66]
```
Compilation succeeded - 1 warning(s)
/usr/bin/sn -q -R ./4.0/bin/Release/HyperFastCgi.exe ./../mono.snk
make[3]: Leaving directory '/HyperFastCgiSrc/src/HyperFastCgi'
make[2]: Leaving directory '/HyperFastCgiSrc/src/HyperFastCgi'
Making all in libnative
make[2]: Entering directory '/HyperFastCgiSrc/src/libnative'
  CC       libhfc_native_la-libev.lo
  CC       libhfc_native_la-fcgi.lo
  CC       libhfc_native_la-fcgi-transport.lo
  CC       libhfc_native_la-socket-helper.lo
  CC       libhfc_native_la-mono-bridge.lo
  CC       libhfc_native_la-mono-bridge-def.lo
  CC       libhfc_native_la-host-list.lo
  CC       libhfc_native_la-domain-bridge.lo
  CCLD     libhfc-native.la
/usr/bin/ld: cannot find -lmono-2.0
collect2: error: ld returned 1 exit status
Makefile:442: recipe for target 'libhfc-native.la' failed
make[2]: *** [libhfc-native.la] Error 1
make[2]: Leaving directory '/HyperFastCgiSrc/src/libnative'
Makefile:369: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/HyperFastCgiSrc/src'
Makefile:405: recipe for target 'all-recursive' failed
make: *** [all-recursive] Error 1
```

**Comment by derFunk** [comment]
Can be fixed by installing libmonosgen-2.0-dev: `apt-get install libmonosgen-2.0-dev`.
See also https://twitter.com/quantumcalc/status/780089322032754688 which I only found too late.

**Comment by xplicit** [comment]
This is mono 4.6 package bug, it is already fixed in sources and should be fixed in repos on the next upload (possible next week)
